### ISSUE TITLE
[FEATURE] Add option to specify extra-prefix for username

### ISSUE THREAD

**Issue by spacecowboy** [zeapo/Android-Password-Store#2215]
### Is your feature request related to a problem? Please describe.

Passwordstore doesn't understand that the first extra line in my password files is the username.

I have been using pass for about 6 years now and I have collected a fair bit of passwords. Pretty much all of my accounts also use a unique email/username. I have formatted the files like this:

    /some/path/SERVICE

where the file looks like this

    PASSWORD
    USERNAME

I have several scripts and some tooling that already depends on this.

* The username is always present as the first extra line
* There is no prefix

### Describe the solution you'd like

I would like an option to manually input the prefix used by the app to identify the username.

Currently the app has a hard coded list of `user:, login:, username:, ....`.

My proposed feature would also solve the potential problem of a prefix which the developers could never have anticipated: for example if someone doesn't type English in their files.

Strictly what I'm proposing is this:

* By default, the custom prefix is set to `default`, (as a value, not literal string), this preserves current behavior with the hardcoded list of prefixes.
* IFF the user has inputted a custom prefix then the following

logic:

* If the prefix is empty: pick the first extra line (this would be what I would use personally)
* If the prefix is non-empty: iterate over the extra lines until such a line is found. If no such line, then nothing is returned.

### Describe alternatives you've considered

I have considered changing all of my password files but it would be tedious so this seems more fun!

### Additional context

If you agree this is a good idea then I am willing to implement the feature myself and submit a PR.

**Comment by spacecowboy** [comment]
Sort of the opposite of https://github.com/android-password-store/Android-Password-Store/issues/1915

**Comment by msfjarvis** [comment]
The feature request may be legitimate but I do not appreciate the tone in the slightest. I'm gonna give you a few hours to go through the issue and fix that.

**Comment by spacecowboy** [comment]
I updated it but I'm not sure I fixed the tone. I am genuinely confused which part gave off a bad tone.

Of course each reader may interpret the text differently but my intent was only to be factual. No judgement or criticism was intended to be conveyed. I think the app is great which why I wanted to contribute.

Been using it too for years but this only cropped up recently because I got a more modern phone and updated to a newer version of Passwordstore which supported Autofill.
